components/admin_console/admin_definition.jsx

```diff
@@ -1104,7 +1104,7 @@ const AdminDefinition = {
                         label_default: 'Amazon S3 Access Key ID:',
                         help_text: t('admin.image.amazonS3IdDescription'),
                         help_text_markdown: true,
-                        help_text_default: '(Optional) Only required if you do not want to authenticate to S3 using an [IAM role](!https://about.mattermost.com/default-iam-role). Enter the Access Key ID provided by your Amazon EC2 administrator.',
+                        help_text_default: '(Optional) Only required if you do not want to authenticate to S3 using an [IAM role](!https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html). Enter the Access Key ID provided by your Amazon EC2 administrator.',
```

Comment:
The same thing goes for many of these ones that are out of our direct control. Those ones I'd be even more worried about since at least we know when we're reorganizing the repos that stuff might break

Reply:
@emdecr - This is a similar case to consider.